buildSrc/src/main/java/org/springframework/boot/build/autoconfigure/DocumentAutoConfigurationClasses.java

```diff
@@ -91,7 +91,7 @@ private void writeTable(AutoConfiguration autoConfigurationClasses) throws IOExc
 				writer.println();
 				writer.printf("| {spring-boot-code}/spring-boot-project/%s/src/main/java/%s.java[`%s`]%n",
 						autoConfigurationClasses.module, autoConfigurationClass.path, autoConfigurationClass.name);
-				writer.printf("| {spring-boot-api}/%s.html[javadoc]%n", autoConfigurationClass.path);
+				writer.printf("| {spring-boot-api}%s.html[javadoc]%n", autoConfigurationClass.path);
```

Comment:
I think we should rather stick to a `/` after the variable rather than no slash. This means that `{spring-boot-api}` would not end with a slash.